src/index.html

```diff
@@ -142,7 +158,52 @@ <h3 class="modal__title">New Post</h3>
         <p class="modal__caption"></p>
       </div>
     </div>
-    <script src="./scripts/validation.js"></script>
-    <script src="./scripts/index.js"></script>
+
+    <div class="modal" id="avatar-modal">
+      <div class="modal__container">
+        <button type="button" class="avatar__close-btn"></button>
+        <h3 class="modal__title">Edit avatar</h3>
+        <form class="modal__form" id="edit-avatar-form">
+          <label for="profile-avatar-input" class="modal__label">
+            Avatar
+            <input
```

Comment:
* Please, add `type="url"` for the `avatar` input, to validate it as a `link` https://snipboard.io/H0Qq26.jpg